### DISCUSSION TITLE
Outlook REST API is deprecated

### DISCUSSION THREAD

**Discussion by weininfuwu** [O365/python-o365#621] [Q&A]
I recentely noticed "Office 365 Exchange Online" has disappered from "API permissions" at Azure portal.
Today I found below web page telling Outlook REST API will terminate service.

https://docs.microsoft.com/en-us/previous-versions/office/office-365-api/api/version-2.0/use-outlook-rest-api

Using O365, I need to send e-mail with large attachment, so I was using MSOffice365Protocol(), which seems to be
using Outlook REST API.

Do you have a plan to allow us to send e-mail with large attachment with MSGraphProtocol()
I really really want you all to do so!!

**Comment by alejcas** [comment]
I have to implement this... but I don't really have time.
It's something fairly easy yo do I think. Maybe someone can send a PR with this.
